tests/install_swiftpm_dependencies/workspace_and_project/Makefile

```
@@ -0,0 +1,3 @@
+generate:
+	cp ../project/project.yml .
+	xcodegen generate
```

Just to be consistent with how you do it in other `Makefile`s:
```suggestion
	xcodegen generate --spec ../project/project.yml
```